playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb

```diff
@@ -0,0 +1,77 @@
+
+<h1 aria-label="online" class="pb_title_kit_1">Opacity</h1>
+<br>
+<%= pb_rails("body", props: {
+  text: "
+  The word opaque is used for describing objects that are fully visible, i.e. with no transparency. 
+  On a web page, a fully opaque element has an opacity value of 1 and a fully transparent element has an opacity value of 0. 
+  By default, all elements have an opacity value of 1. From 0 to 1, the opacity property controls the transparency level of the element."
+}) %>
+<br>
+<h4 aria-label="online" class="pb_title_kit_4 pt_sm">Opacity Tokens</h4><br>
+<%= pb_rails("body", props: {
+  text: "Make your own styles using Playbook tokens to keep your site consistent. To use the opacity for your custom css import into your scss file:"
+}) %><br>
+
+<%= pb_rails("card", props: {shadow: "deep", margin_bottom: "xl", padding: "none", padding_top: "md" }) do %>
+<ul class="opacity-play pb--utlities-color">
+<li data-name="Data 1" data-variable="$data_1">
+<div class="pb--color_data_1 opacity_1"></div>
+<p class="name">Opacity_1</p>
+<p class="var">$opacity_1</p>
+</li>
+<li data-name="Data 1" data-variable="$data_1">
+<div class="pb--color_data_1 opacity_2">
+</div>
+<p class="name">Opacity_2</p>
+<p class="var">$opacity_2</p>
+</li>
+<li data-name="Data 1" data-variable="$data_1">
+<div class="pb--color_data_1 opacity_3">
+</div>
+<p class="name">Opacity_3</p>
+<p class="var">$opacity_3</p>
+</li>
+<li data-name="Data 1" data-variable="$data_1">
+<div class="pb--color_data_1 opacity_4">
+</div><p class="name">Opacity_4</p>
+<p class="var">$opacity_4</p>
```

Comment:
You have 0 indentations here. I can't tell what is a container. Please correct this whole file.

Tab to indent
Shift + Tab to outdent